bin/scriptoni.js

```diff
@@ -20,13 +22,11 @@ function execSync(command, options) {
   logger.bin('executing: ', command);
   return _execSync(command, options);
 }
-var execSyncOptions = { cwd: process.cwd(), encoding: 'utf-8' };
-/*
- * clean /build folder
- */
+
 function clean() {
-  execSync('rm -rf build/*', execSyncOptions);
-  execSync('mkdir -p build', execSyncOptions);
+  logger.bin('clean /build folder')
+  rimraf.sync('build');
+  fs.mkdirSync('build');
```

Comment:
are you sure these `build` paths are still OK? we were passing a `cwd` to exec before, depending on the cwd of the caller of scriptoni.. not sure how `rimraf.sync` and `mkDirSync` behave? probably already ok?